cmd/cluster-operator-controller-manager/app/options/options.go

```diff
@@ -105,6 +106,7 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet, allControllers []string, disabled
 	fs.Int32Var(&s.ConcurrentMachineSetSyncs, "concurrent-machine-set-syncs", s.ConcurrentMachineSetSyncs, "The number of machine set objects that are allowed to sync concurrently. Larger number = more responsive machine sets, but more CPU (and network) load")
 	fs.Int32Var(&s.ConcurrentMachineSyncs, "concurrent-machine-syncs", s.ConcurrentMachineSyncs, "The number of machine objects that are allowed to sync concurrently. Larger number = more responsive machines, but more CPU (and network) load")
 	fs.Int32Var(&s.ConcurrentMasterSyncs, "concurrent-master-syncs", s.ConcurrentMasterSyncs, "The number of master machine objects that are allowed to sync concurrently. Larger number = more responsive master machines, but more CPU (and network) load")
+	fs.Int32Var(&s.ConcurrentAcceptSyncs, "concurrent-accept-syncs", s.ConcurrentAcceptSyncs, "The number of cluster objects that are allowed to accept concurrently. Larger number = more responsive accept jobs, but more CPU (and network) load")
```

Comment:
s/cluster/machine set/ since the accept controller is syncing on machine sets now.